packages/vector_graphics_compiler/bin/vector_graphics_compiler.dart

```diff
@@ -5,6 +5,13 @@ import 'package:args/args.dart';
 import 'package:vector_graphics_compiler/vector_graphics_compiler.dart';
 
 final ArgParser argParser = ArgParser()
+  ..addOption(
```

Comment:
Why not just use LD_Library_path?

Reply:
Not a thing on windows, probably not a thing in g3, plus don't want to copy around a bunch of binaries - and plus plus we need to turn this on behind a flag and so the option doubles up

Reply:
I'm assuming g3 will be building this and it'll get taken care of being put in the right spot as a dep.

On Windows it'd be `PATH` or the current directory.

It feels very strange to pass this in as a command line argument, especially since it'll vary by OS/environment. I guess I'm fine with making it an option, but maybe making it a hidden one - I don't see this sticking around long term.

Reply:
Yeah, I definitely agree with you here - I'd think this is temporary so we can merge this without needing to continually rebase the PR for testing.